adapters/adtelligent/adtelligent_test.go

```diff
@@ -0,0 +1,548 @@
+package adtelligent
+
+import (
+	"encoding/json"
+	"github.com/mxmCherry/openrtb"
+	"github.com/prebid/prebid-server/adapters"
+	"github.com/prebid/prebid-server/openrtb_ext"
+	"net/http"
+	"strings"
+	"testing"
+)
+
+func TestAdtelligentAdapterInfo(t *testing.T) {
+	an := NewAdtelligentAdapter(adapters.DefaultHTTPAdapterConfig)
+
+	name := an.Name()
+	if name != "Adtelligent" {
+		t.Errorf("Name '%s' != 'Adtelligent'", name)
+	}
+
+	familyName := an.FamilyName()
+	if familyName != "adtelligent" {
+		t.Errorf("FamilyName '%s' != 'adtelligent'", familyName)
+	}
+
+	skipNoCookies := an.SkipNoCookies()
+	if skipNoCookies != false {
+		t.Errorf("SkipNoCookies should be false")
+	}
+}
+
+func TestAdtelligentOpenRTBRequest(t *testing.T) {
```

Comment:
Can these tests be migrated to the JSON format? From [the docs](https://github.com/prebid/prebid-server/blob/master/docs/developers/add-new-bidder.md#automated-tests):

> Since most Bidders communicate through HTTP using JSON bodies, you should use the [JSON-test utilities](https://github.com/prebid/prebid-server/blob/master/adapters/adapterstest/test_json.go). This comes with several benefits, which are described in the source code docs.

Check out [the appnexus test files](https://github.com/prebid/prebid-server/tree/master/adapters/appnexus/appnexustest) for some examples.

You can basically replace this whole test file with:

> func TestJsonSamples(t *testing.T) {
>     adapterstest.RunJSONBidderTest(t, "adtelligenttest", NewAdtelligentBidder())
> }

The benefits are described [here](https://github.com/prebid/prebid-server/blob/master/adapters/adapterstest/test_json.go#L15). I should probably update this comment; although it says "the project doesn't require it," what it means is "the project doesn't require it if your Bidder can't use this format (e.g. you don't speak JSON to the external server)".

Reply:
Great tools :) Yes, I did it.